operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -1299,6 +1299,15 @@ restorAppAndLibs() {
 
 }
 
+#
+restoreZippedDbWallets() {
+  local count=$(find ${DOMAIN_HOME}/wlsdeploy/dbWallets/*/*.zip -type f 2>/dev/null | wc -l)
+  if [ $count > 0 ] ; then
+    find ${DOMAIN_HOME}/wlsdeploy/dbWallets/*/*.zip -type f  | xargs -I % sh -c 'unzip -jo % -d $(dirname %) ; rm %'
+  fi
```

Comment:
Thoughts/issues:
- The ">" isn't a valid comparator, it'll return true even if count is 0. Fix by using '-gt'.
- It's a little safer/cleaner to use `"$count"` instead of plain `$count`. 
- The function keeps going even if the count expression fails. Fix by appending ' || return 1'.